superflore/generate_installers.py

```diff
@@ -24,14 +23,14 @@
 
 
 def generate_installers(
-    distro_name,             # ros distro name
+    distro,                  # ros distro
     overlay,                 # repo instance
     gen_pkg_func,            # function to call for generating
     preserve_existing=True,  # don't regenerate if installer exists
     *args,                   # any additional args for gen_pkg_func
     **kwargs                 # any additional keyword arguments
 ):
-    distro = get_distro(distro_name)
+    distro_name = distro._distribution_file.name
```

Comment:
It's better not to use the "private" attribute prefixed by underscore here. You can just use distro.name. 

All the elements are proxied through: https://github.com/ros-infrastructure/rosdistro/blob/e03a8687d7d4253fad08e4ff80e7c0f5c8fb1984/src/rosdistro/distribution.py#L62

```suggestion
    distro_name = distro.name
```

Reply:
Great, thank you!